db/seeds.rb

```diff
@@ -12,8 +12,8 @@ def load_data
     load_crops
     load_plant_parts
 
-    # for development environments only
-    if Rails.env.development?
+    # for development and staging environments only
+    if Rails.env.development? || Rails.env.staging?
```

Comment:
That makes sense. We should also run `load_products`, right? I've made [PR 627](https://github.com/Growstuff/growstuff/pull/627) for this.